pkg/sql/catalog/lease/kv_writer.go

```diff
@@ -61,19 +63,25 @@ func leaseTableWithID(id descpb.ID) catalog.TableDescriptor {
 func (w *kvWriter) versionGuard(
 	ctx context.Context, txn *kv.Txn,
 ) (settingswatcher.VersionGuard, error) {
-	return w.settingsWatcher.MakeVersionGuard(ctx, txn, clusterversion.V23_1_SystemRbrCleanup)
+	return w.settingsWatcher.MakeVersionGuard(ctx, txn, clusterversion.V23_2)
 }
 
 func (w *kvWriter) insertLease(ctx context.Context, txn *kv.Txn, l leaseFields) error {
```

Comment:
I read about the previous migration we did to change `system.lease` table from a regional-by-table to a regional-by-row table. I saw it uses 4 cluster versions to achieve this: dual-write, backfill-new, read-new-only, delete-old. 

We are now doing yet another upgrade to the `system.lease` table (of adding a new column `sessionID`). Why don't we follow the same pattern of stepping through 4 cluster versions?